mysql-test/t/sp-bugs.test

```diff
@@ -476,3 +476,31 @@ SHOW CREATE FUNCTION очень_очень_очень_очень_очень_оч
 DROP FUNCTION очень_очень_очень_очень_очень_очень_очень_очень_длинная_строка_é;
 
 SET NAMES default;
+
+--echo #
+--echo # PS-6094 - Handler fails to trigger on Error 1049 or SQLSTATE 42000 or plain sqlexception
```

Comment:
Use upstream bug ids for upstream bugs: Bug#97682

Reply:
done